lib/parameters.py

```diff
@@ -50,11 +50,11 @@ def gen_avc_cqp_variants(spec, profiles):
       for profile in cprofiles:
         yield [
           case, variant["gop"], variant["slices"], variant["bframes"],
-          variant["qp"], variant["quality"], profile
+          variant["qp"], variant["quality"], profile, variant.get("bdepth", 0)
```

Comment:
ffmpeg-vaapi default bdepth is 1.  Therefore, this could change the test behavior for existing use config cases that don't have bdepth defined.